### ISSUE TITLE
Im so lost.

### ISSUE THREAD

**Issue by deanfourie1** [dwainscheeren/dwains-lovelace-dashboard#828]
I dont get this dashboard, 

I cant help but think that im doing something wrong, but I cant do anything.

1. cant change the amount of rows displayed on different devices.
2. cant move cards around, the only cards I can move around are the cards selected from below the dashboard, when adding a fresh new card from the top, I cannot move the position of the card???
3. I seem to be unable to edit the chips above the area cards?
4. also, more of a bug, but when there is nothing below the page, it is still scrollable. its just empty space and should not be scrollable.

It just seems to be really Un customizable and un user friendly for editing. I really want to use it because i like the idea of the area cards, but its a PITA to work with for me right now.

**Comment by ghost** [comment]
> I dont get this dashboard,
> 
> I cant help but think that im doing something wrong, but I cant do anything.
> 
> 1. cant change the amount of rows displayed on different devices.t


Agreed. You can control how wide an object can be, depending on screen format but the row column setting seams dead. Never got that to change anything either 
> 2. cant move cards around, the only cards I can move around are the cards selected from below the dashboard, when adding a fresh new card from the top, I cannot move the position of the card???

Automatically added cards can be changed position of relative of eachother and if you choose to section by type, only relative to other in the same device type.
Manually added cards can only be added on top of page or bottom of page.
There is a way around this.
You can let a meaningless entity, in the correct section, stay exposed and simply replace the card for that meaningless entity (eg a tracker) with whatever you want. A completely different card or bundle of cards.
I prefer to build bundles, with stacks, in another, standard HA dash and simply copy paste. The stack for my boiler is built that way and replaces one single entity. (Pic below)
![Screenshot_20240821_131423](https://github.com/user-attachments/assets/072c66ab-b087-4d14-9e79-83822025d057)

> 3. I seem to be unable to edit the chips above the area cards?

Think it's not editable, not sure
> 4. also, more of a bug, but when there is nothing below the page, it is still scrollable. its just empty space and should not be scrollable.
> 
> It just seems to be really Un customizable and un user friendly for editing. I really want to use it because i like the idea of the area cards, but its a PITA to work with for me right now.

The point of this dash is not to be a tool for extreme customisation. It is built to be a dash that looks ok in every screen format so you only need one dash that works for all format and to be a dash that builds itself and will keep doing that even if you do changes and most those changes CAN be done via gui.
Perfect for beginners or lazy f.ckers as myself that hate spending time on frontend.
So, if you are the kind of user that needs to tweek everything, Dwain's is not for you.
To make use of Dwain's, you have to be able to find yourself at home in its structure and value its benefits, realising you can't change everything.

**Comment by deanfourie1** [comment]
Hmm yea ok good to know its not me then, just a shame because it would be a really good dashboard if it had a few more basic functions.

I will try make it work, but I end up spending more time trying to arrange my cards etc then I should.  There are not many options for a similar concept with the changing panels based on the area.

**Comment by ghost** [comment]
There are always room for improvements but the basic functionality is there and will allow you to skip building and maintaining several dashboards for different use cases.
My family appreciate that the gui is the same,  wherever they try to reach HA, from their mobiles or tablets, from their laptops or from our wall mounted dashboards. Also eliminates the risk that some devices are left out from any of these dashboards by mistake.
I don't know how your smart home works but mine had definitely not gotten to the the state where it is ready enough to build static dashboards for different purposes. Would only be a matter of hours before they start to drift apart on how "up to date" they are.
I gladly sacrifice some customization possibilities to have something that works for the entire family with a minimal invested workload for me. I have better things to do then to maintaining dashboards.

**Comment by dwainscheeren** [comment]
Thanks for reporting this.

Dwains Dashboard 3.9.0 has now been released, and the source files for DD3 are available in this repository.

DD3 is no longer under active feature development and will slowly be phased out. Active development continues in Dwains Dashboard Next:
https://github.com/dwainscheeren/dwains-dashboard-next

At the time of writing, Dwains Dashboard Next is still in alpha.

If this issue is a feature request, please check whether it fits Dwains Dashboard Next instead.

If this issue is a bug, please retest with DD3 version 3.9.0. If the bug is still reproducible there, please open a new issue with:

- Home Assistant version
- Dwains Dashboard version
- browser and device
- browser console errors
- clear reproduction steps

For more context, see #906.

For that reason, this older issue is being closed as part of the DD3 issue tracker cleanup.
